tests/datacatalog/app_test.py

```diff
@@ -27,7 +27,7 @@ class DatasetTestCase(BaseTestCase):
         ]:
             response = await self.client.request("GET", endpoint)
             text = await response.text()
-            self.assertEquals(response.status, 200)
+            self.assertEqual(response.status, 200)
```

Comment:
Deze lelijke `assertEqual` statements herinneren me eraan dat we van dat lelijke test-framework af moeten. Maar niet nu.